src/Server.php

```diff
@@ -16,14 +16,26 @@ public function __construct(LoopInterface $loop)
         $this->loop = $loop;
     }
 
-    public function listen($port, $host = '127.0.0.1')
+    public function listen($port, $host = '127.0.0.1', $streamContext = array())
     {
+        if (isset($streamContext['ssl']) && PHP_VERSION_ID < 50600) {
+            throw new \RuntimeException(
+                'Secure connections are not available before PHP 5.6.0'
```

Comment:
This is not strictly true, but I haven't been able to get it working with PHP 5.5.x using a simple HTTP server (and select loop) and web browser.  The problem seems to be that select() will permit a read of the first character (The 'G' in 'GET / HTTP/1.1') and then no more; so the http server doesn't send a response.  Works fine with wget though.  More work needed to work out what's going wrong.